### DISCUSSION TITLE
Fine tune arguments to learn new knowledge without forgetting previous

### DISCUSSION THREAD

**Discussion by SutirthaChakraborty** [mlfoundations/open_clip#996] [Q&A]
What should be the fine-tuning argument to make sure we are just adding knowledge to the existing pretrained model, without affecting their past training and learn few of my new words with that ? 

When I am retraining, it's  forgetting the pretrained vocab. I uses small learning rate and wd. 
Do anyone has the perfect argument values to do that ?

**Comment by Kimsure** [comment]
Continual Learning might help you

**Reply by SutirthaChakraborty** [reply]
how do we do that?
